alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetAlertsAction.kt

```diff
@@ -169,10 +182,8 @@ class TransportGetAlertsAction @Inject constructor(
                     alertIndex = getMonitorResponse.monitor!!.dataSources.alertsIndex
                 }
             }
-        return if (alertIndex == AlertIndices.ALERT_INDEX)
-            AlertIndices.ALL_ALERT_INDEX_PATTERN
-        else
-            alertIndex
```

Comment:
cant get reverted. This will cause normal alerts (non-SA alerts) to be affected

Reply:
ack

Reply:
reverted